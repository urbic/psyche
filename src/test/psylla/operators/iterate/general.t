2. { dup 1. exch div sub } iterate 10 limited {?} forall
