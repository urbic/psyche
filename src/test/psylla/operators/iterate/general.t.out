2.0
1.5
0.8333333333333334
-0.3666666666666666
2.3606060606060613
1.9369860349321204
1.4207200516128116
0.7168516161213909
-0.6781372177053588
0.7964905919638134
